VenuApp/src/main/resources/templates/welcome/welcomePage.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <base href="./"/>
+    <meta charset="UTF-8"/>
+    <title>VENU</title>
+    <link href="./VenuApp/src/main/resources/templates/welcome/welcome.css" rel="stylesheet" />
+</head>
+<body>
+    <header>
+        <h1 class="title" style="color: blue">Welcome Infinity Code Crew!</h1>
+    </header>
+    <div>
+        <img src="/VenuApp/src/main/resources/images/concertbg.png" alt="concert stage with a crowd" />
+        <p>test</p>
+    </div>
+</body>
+</html>
```

Comment:
this html page is removed in a future commit, I was testing putting css directly in an html page to see if my stylesheet wasn't loading in or if it was another issue